.github/workflows/pr-checks.yaml

```diff
@@ -29,3 +29,9 @@ jobs:
           fetch-depth: 0
       - name: Check if dockerimage build is working
         run: docker build -f ./Dockerfile .
+
+      - name: Setup Container and run Conftest Unittest
+        uses: ./ # Uses an action in the root directory
+        id: test_image
+        with:
+          image: 'quay.io/cvpops/test-index:v4.10'
```

Comment:
I've created the `quay.io/redhat-appstudio/sample-image:test-index-pass` image that we can use instead of this one.

Reply:
Thanks